pkg/config/node.go

```diff
@@ -261,9 +263,15 @@ func GetConfig(kubeconfigPath, clusterConfigPath, resolvConfPath string, apiVip
 	return node, err
 }
 
+// getSortedBackends build config to communicate with kube-api based on kubeconfigPath parameter value, if kubeconfigPath is not empty it will build the
```

Comment:
Ick, I dislike overloading variable meanings this way, but I guess client-go already did the same thing so we might as well go with it. :-/